packages/node/src/policy-util.js

```diff
@@ -11,21 +11,38 @@
 import { jsonStringifySortedPolicy, mergePolicy } from 'lavamoat-core'
 import nodeFs from 'node:fs'
 import nodePath from 'node:path'
+import { toKeypath } from 'to-keypath'
```

Comment:
nice one, but let's vendor it. No need to have a prod dependency and the risk is always there.